### PR TITLE
Fix: TreePicker remove parent node on child added

### PR DESCRIPTION
**Pull request by camjc**
- Adding a parent removes all children (already),
  and adding a child removes all parents (now).

### CODE REVIEW

**Comment by vinteo** [comment]
LGTM

File: src/components/adslotUi/TreePickerComponent.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ import TreePickerPropTypes from 'helpers/propTypes/TreePickerPropTypes';
 import TreePickerPure from 'components/adslotUi/TreePickerPureComponent';
 import React, { PropTypes } from 'react';
 
+const idInPath = ({ id, path }) => _(path).map('id').contains(id);
```
Comment (etaoins) [inline src/components/adslotUi/TreePickerComponent.js]: Maybe `idIsInPath`? The current name doesn't read like a predicate

**Comment by etaoins** [comment]
LGTM

**Comment by vinteo** [comment]
Merging...

**Pull request merged**